internal/testutil/tsbaseline/js_emit_baseline.go

```diff
@@ -104,7 +104,11 @@ func DoJsEmitBaseline(
 					expected := original.Content
 					actual := doc.Content
 					var patch strings.Builder
-					diff.Text("Expected\tThe full check baseline", "Actual\twith noCheck set", expected, actual, &patch)
+					err := diff.Text("Expected\tThe full check baseline", "Actual\twith noCheck set", expected, actual, &patch)
```

Comment:
t.Fatal here, or ignore the error and skip the test (if the file is too big for example)